### PR TITLE
Cleaning up and organizing Less files

### PR DESCRIPTION
**Pull request by egid**
We had an unused file and a bunch of unused / deprecated styles from the hackathon rush. This PR goes through and cleans up styles with (in most cases) no change to the end result.

I also fixed a couple of minor design bugs.

### CODE REVIEW

**Comment by gigabo** [comment]
Awesome.  Thanks @egid!

File: packages/react-server-website/components/doc-contents.less [hunk 1]
```diff
@@ -15,23 +15,22 @@
 	@media (min-width: @min-medium-screen) {
 		margin: 0;
 
-		box-shadow: 0px 1px 3px 0px rgba(0, 0, 0, 0.25);
 		border: 1px solid @border-color;
 		border-radius: 2px;
+		.default-box-shadow()
```
Comment (decoy31) [inline packages/react-server-website/components/doc-contents.less]: Missing semicolon.

File: packages/react-server-website/middleware/Theme/button.less [hunk 2]
```diff
@@ -7,7 +7,7 @@ button {
 .Button {
 	display: block;
 	text-decoration: none;
-	color: #FFFFFF;
+	color: white;
```
Comment (decoy31) [inline packages/react-server-website/middleware/Theme/button.less]: Was this change just a personal preference of yours or is there an additional benefit I don't know about?

File: packages/react-server-website/middleware/Theme/reset.less [hunk 3]
```diff
@@ -1,62 +0,0 @@
-/* http://meyerweb.com/eric/tools/css/reset/
```
Comment (decoy31) [inline packages/react-server-website/middleware/Theme/reset.less]: Why are you removing the reset?

Comment (egid) [inline packages/react-server-website/middleware/Theme/reset.less]: We weren't ever actually using it!

**Pull request merged**